lib/tpl/dokuwiki/main.php

```diff
@@ -16,7 +16,7 @@
 <html lang="<?php echo $conf['lang'] ?>" dir="<?php echo $lang['direction'] ?>" class="no-js">
 <head>
     <meta charset="utf-8" />
-    <title><?php tpl_pagetitle() ?> [<?php echo strip_tags($conf['title']) ?>]</title>
```

Comment:
I notice that you remove `$conf['title']` from the title tag, which is a big change. Do you mind explaining why this is preferred?

Reply:
I use Translator plugin and actually have :start page with title same as my dokuwiki title, because it is shown also in bread crumbs navigation.